src/test/config.sol

```diff
@@ -1955,7 +1955,7 @@ contract Config {
             flipper_mom:  0,
             dog_hole:     2 * MILLION,
             clip_buf:     120_00,
-            clip_tail:    140 minutes,
+            clip_tail:    8_400,
```

Comment:
I prefer minutes here, it is what we have in other `config`s and double checks our second number in the spell

```suggestion
            clip_tail:    140 minutes,
```